scripts/staking/config/tbos_snapshot_staking.config.ts

```diff
@@ -0,0 +1,12 @@
+export const TBOS_SNAPSHOT_STAKING_CONFIG = {
+  BOB: {
+    averageBlockTime: 15,
+    stakingAddress: "0xc17C6462cEAFE9A8819258c6bA168BEF5544Fc21",
+    rpcUrl: "https://rpc.gobob.xyz/",
+  },
+  RSK: {
+    averageBlockTime: 30,
+    stakingAddress: "0x5684a06CaB22Db16d901fEe2A5C081b4C91eA40e",
+    rpcUrl: "https://mainnet-dev.sovryn.app/rpc",
+  },
+} as const;
```

Comment:
averageBlockTime is not needed anymore
```suggestion
export const TBOS_SNAPSHOT_STAKING_CONFIG = {
  BOB: {
    stakingAddress: "0xc17C6462cEAFE9A8819258c6bA168BEF5544Fc21",
    rpcUrl: "https://rpc.gobob.xyz/",
  },
  RSK: {
    stakingAddress: "0x5684a06CaB22Db16d901fEe2A5C081b4C91eA40e",
    rpcUrl: "https://mainnet-dev.sovryn.app/rpc",
  },
} as const;
```